env.ts

```diff
@@ -8,6 +8,7 @@ export interface IEnvironment {
   OROCHI_PUBLIC_KEY: string;
   OROCHI_OWNER: string;
   OROCHI_OPERATOR: string;
+  WALLET_PRIVATE_KEY: string;
```

Comment:
We have the passphrase why do we need `WALLET_PRIVATE_KEY`?

Reply:
Tks. Modified!